views/base.njk

```diff
@@ -42,6 +42,19 @@
 
     <meta name="viewport" content="width=device-width, initial-scale=1">
 
+    <!-- For Facebook -->
+    <meta property="og:title" content="Federalist compliantly hosts federal government websites." />
+    <meta property="og:type" content="article" />
+    <meta property="og:image" content="/images/social-share/facebook-1200x627.png" />
+    <meta property="og:url" content="https://federalist.18f.gov" />
+    <meta property="og:description" content="Federalist expertly manages the backend work of deploying secure, compliant, and scalable government websites, allowing federal offices to focus on their content." />
+
+    <!-- For Twitter -->
```

Comment:
It looks like several of the the `twitter:` tags will fallback to `og:` counterparts, so maybe we should just specify the ones we want to be different. If I'm reading the [docs](https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup) correctly, I think we can leave off `twitter:title` and `twitter:description` since they have the same content in their `og:` counterparts.

docs: https://developer.twitter.com/en/docs/tweets/optimize-with-cards/overview/markup

Reply:
Ah! yes, forgot that it defaults to the OG tags. Removed extra meta tags in 1964bfa